Html pages rafactoring

ddavlet opened the pull request:
pages styles now depend on css. If we want to change styling, we can change it in CSS. It does work for all pages

olympiaobr reviewed (dismissed):
Are you supposed to land at the page below when you just click upload without a chosen file for uploading?

 body { font-family: Arial, sans-serif; background-color: #f4f4f4; color: #333; margin: 0; padding: 0; } header { background-color: #4CAF50; color: white; padding: 20px; text-align: center; } main { padding: 20px; } footer { background-color: #333; color: white; text-align: center; padding: 10px; position: fixed; bottom: 0; width: 100%; }
File uploaded!
Index of ./web/upload/

ddavlet commented:
1. preceding style info:
"body { font-family: Arial, sans-serif; background-color: #f4f4f4; color: #333; margin: 0; padding: 0; } header { background-color: #4CAF50; color: white; padding: 20px; text-align: center; } main { padding: 20px; } footer { background-color: #333; color: white; text-align: center; padding: 10px; position: fixed; bottom: 0; width: 100%; }"

fixed in other pull request. 

2. Regarding printing the list of uploads instead of returning some error:
  I was thinking about it, but didn't find appropriate error page for it. On the one hand it is bad request, on the other hand 
  it is not that bad to return 400 code. So I just decided to handle it as normal upload attempt, but it can be a subject for 
  discussion
